project_timeline/views/project_task_view.xml

```diff
@@ -59,8 +59,8 @@
         <field name="type">timeline</field>
         <field name="arch" type="xml">
             <timeline
-                date_start="date_assign"
-                date_stop="date_end"
+                date_start="planned_date_start"
+                date_stop="planned_date_end"
```

Comment:
@victoralmau @pedrobaeza these changes are causing an error in the model `report.project.task.user` because the fields do not exist there. Please review. I reverted it in the migration to 17. I believe the fields should be added to the model first before including them in the view.
https://github.com/OCA/project/pull/1354/commits/994622708ce158aededd5516b15c6895ac94700e#diff-f2f81dcbaa2dc0db56e12f2ca784d921c1acbd33e73500563c4c07a455808fc2R65

![image](https://github.com/user-attachments/assets/0d3f3165-1df1-4d26-b0ab-34088c2eead7)

Reply:
Please fix it here and don't revert in 17